src/libraries/StableOracleMath.sol

```diff
@@ -1,7 +1,7 @@
 // TODO: License, this was taken from Balancer right?
 pragma solidity ^0.8.0;
 
-import { FixedPointMathLib } from "solmate/utils/FixedPointMathLib.sol";
+import { FixedPointMathLib } from "solady/utils/FixedPointMathLib.sol";
```

Comment:
Would be good to benchmark our `sqrt` implementation (can also be in another PR).

Reply:
I kinda tested it previously when addressing the comment to use a more efficient `Math` lib. Our naive lib with the Babylon impl is definitely a lot less efficient than theirs

Reply:
are we using solady sqrt or balancer sqrt?

Reply:
eh? long time ago we were using balancer's Babylonian method. Then we switched to solmate's which was a lot more efficient. Now it seems that solady's one is even more efficient. 

But if you're talking about comparing the correctness, we have the our unit tests using the Babylonian impl, checking against the src's use of whatever impl of `sqrt`

Reply:
Sounds good, happy to resolve